microsoft/botframework-solutions
# [To Do] update luis model and fix an account related issue

Update To Do luis model and fix an account related bug
## Description
1. Optimize LU model of ToDo skill
2. Close #873 

### Architecture

### Bug Fixes
Fix bug #873

### Conversational Experience

### Maintenance

### Language Understanding
1. Optimze the en-us model based on the latest LU model. 
2. Add back intents ShowNext/ShowPrevious in ToDo LU model.
2. Apply the latest zh-cn model from LU team.

## Testing Steps
1. Deploy the latest LU model and try to ask "show more to do items" to see if it works correctly.
2. Apply for a new outlook account(do nothing with the account) and use this account to login VA and trigger ToDo skill. Before the bug fix, it only shows "Sth went wrong". After fixed, it will prompt "Sorry, it looks like something went wrong with your email account. Please follow the instructions in your Inbox to verify your account." for this specific account not activated issue.

## Checklist
